pkg/ruler/ruleset.go

```diff
@@ -281,8 +281,9 @@ func GenerateInTotoLink(reports []Report, fileBytes []byte) in_toto.Metablock {
 
 	materials := make(map[string]interface{})
 	request := make(map[string]interface{})
+	fileBytes = append(fileBytes, 10) // FIXME: we need an additional newline here?!
 	request["sha256"] = fmt.Sprintf("%x", sha256.Sum256([]uint8(fileBytes)))
-	materials["request"] = request
+	materials["deployment.yml"] = request
```

Comment:
Should this be hardcoded or the value of the file passed in?
The file passed in wont always be called "deployment.yml"

Reply:
Yeah. Unfortunately, for the request it only sees a payload without a filename. i wonder if we can encode the filename as a GET parameter. What's your take on it?

Reply:
I think kubesec's default file name for api submissions is either `deployment.yaml` or `.yml` somewhere

Should be able to grab the file name when used from cli

Reply:
Agreed. We can set a default, then for CLI: get filename if arg. Use default if `stdin`. URL: use default, unless override value set. 

A single file may contain multiple YAML documents etc.

Reply:
To complete in another PR